lesswrong.com/Rationality: A-Z/05_Mere_Goodness/02_Value_Theory/06_2-Place_and_1-Place_Words/content.html

```diff
@@ -1,4 +1,4 @@
-<p><a onclick="window.open(this.href, '_blank', 'width=300,height=406,scrollbars=no,resizable=no,toolbar=no,directories=no,location=no,menubar=no,status=no,left=0,top=0'); return false" href="../../../../book.english/img/2f0d638ee6de765ac7d0b7467447e08e.jpg"><img style="margin: 0px 0px 5px 15px; float: right; width: 100px; height: 134px;" title="Monsterwithgirl_2" src="../../../../book.english/img/2f0d638ee6de765ac7d0b7467447e08e.jpg" border="0" alt="Monsterwithgirl_2" /></a></p>
```

Comment:
book.english or book.en?

Reply:
I am trying to find "book.english" in the repository, but no luck. Try to regenerate docbooks (make valid_book), this looks like HTML-file generated from old dbk file.

Reply:
New directory is book.en. It's removed part of content